### PR TITLE
fix(SystemPolicyCards): RHICOMPL-1213 same height on all cards

### PR DESCRIPTION
**Pull request by vkrizan**
sets the height to 100% to fill the cell in a grid through newly
accessible prop.

### CODE REVIEW

**Review by akofink — COMMENTED** [review]
Well, the cards are always the same height, but the layout inside the cards seems... a bit off sometimes:
image

File: packages/inventory-compliance/src/SystemPolicyCard.js [hunk 1]
```diff
@@ -133,7 +133,8 @@ SystemPolicyCard.propTypes = {
         policyType: PropTypes.string,
         compliant: PropTypes.bool,
         ssgVersion: PropTypes.string,
-        supported: PropTypes.bool
+        supported: PropTypes.bool,
+        style: PropTypes.string
```
[the remark this answers is not in the document]
Comment (vkrizan) [reply packages/inventory-compliance/src/SystemPolicyCard.js:137]: Kinda, but unfortunately not for all cases, as I've just found out. I guess the only builtin one is `key`. `style` is an HTML prop that might work out of the box, but the component is not mapped (extends) to any HTML component and the style has to be passed down to `Card` and in the end into `<article>`.

**Comment by vkrizan** [comment]
@akofink thanks for testing it and finding the issue with the warning. I've fixed it by placing it onto `CardFooter` with some CSS changes. (See new commit)

**Comment by vkrizan** [comment]
I've also refactored it to use PF's `Alert`.

**Review by akofink — APPROVED** [review]
Thanks, @vkrizan! Works as expected.

**Pull request merged**